_posts/2021-12-23-JabRef5-4.md

```diff
@@ -0,0 +1,38 @@
+---
+title:  🎄 JabRef 5.4 Release 🎄
+id: jabref-5-4
+author: "[JabRef Maintainers](https://github.com/JabRef/jabref/blob/main/MAINTAINERS)"
+bg: jabref-font
+color: white
+---
+
+We are proud to announce the release of the next milestone of JabRef, this time JabRef 5.4, just in time for the holidays!
+It's been roughly half a year since the previous release of 5.3, and we are happy to announce that despite the still ongoging Covid-19 waves we were all fortunately able to continue working on JabRef.
+
+## Notable Changes and New Features
+
+Thanks to this year's Google Summer of Code, JabRef now as a PDF-fulltext search and improved metadata support. Search across libraries and in your linked pdf files.
```

Comment:
Missing subject and verb in "Search across libraries and in your linked pdf files."?